src/store/groups.js

```diff
@@ -88,6 +108,12 @@ const mutations = {
 		const index = contacts.findIndex(search => search === contact.key)
 		if (index > -1) {
 			contacts.splice(index, 1)
+			if (state.orderGroups === 'amount') {
```

Comment:
I don't understand this part though, could you add a comment and detail it to me please? :)